parabolic_adjustment_factors:
    - 0.017248
    - 0.018778
    - -0.000216
    - 0.019631
    - 0.847335